docs/components/tree-list.md

```diff
@@ -215,6 +215,26 @@ Boolean to hide icons. Defaults to `false`.
 <KTreeList hide-icons :items="items" />
 ```
 
+### collapsible
+
+Boolean to enable/disable collapse feature. Defaults to `false`.
+
+<KTreeList :collapsible="true" :items="collapsibleItems" />
```

Comment:
can you update the example to also include an item with no children as the second item so we can see how it displays alongside the collapsible items?